kubernetes/.openapi-generator-ignore

```diff
@@ -4,6 +4,7 @@
 *sbt*
 git_push.sh
 README.md
+pom.xml
```

Comment:
the default behavior of the underlying code-generator is to omit the pom.xml file when it's already present, so adding this the list won't break anything..

Reply:
oops, my bad. I asked him to do this...